lib/cog/bundle/embedded.ex

```diff
@@ -63,6 +63,8 @@ defmodule Cog.Bundle.Embedded do
     Config.gen_config(Cog.Util.Misc.embedded_bundle,
                       "Core chat commands for Cog",
                       version,
+                      "Operable <support@operable.io>",
```

Comment:
This is cool, but we should probably simplify our config generation code soon; this is literally the only bundle that uses it, so we don't need to take so many pains to keep it general.